src/impl/EspNowNode.cpp

```diff
@@ -84,8 +87,9 @@ bool EspNowNode::setup() {
   ESP_ERROR_CHECK(esp_wifi_init(&cfg));
   ESP_ERROR_CHECK(esp_wifi_set_storage(WIFI_STORAGE_RAM));
   ESP_ERROR_CHECK(esp_wifi_set_mode(WIFI_MODE_STA));
+  ESP_ERROR_CHECK(esp_wifi_set_ps(WIFI_PS_NONE));
```

Comment:
Why is this one needed?

Reply:
That makes sure wifi doesn't go into power saving mode, which can result in it not receiving ESP-NOW frames. However, given the node wakes up gor short period of time, there probably isn't time for power saving to kick in on the node, so this could be removed. It should stay on the host though.

Reply:
I've removed it given my comment above.